README.md

```diff
@@ -1,49 +1,47 @@
 # CPAN.org content repository
 
-This repository is for maintaining the non-source-code, non-archive
-content on cpan.org.  Discussion related to this should be on the
-[cpan-workers mailing list][cpan-workers].
+This repository is for maintaining the non-source-code, non-archive content on [cpan.org](https://www.cpan.org).
+Discussion related to this should be on the [**cpan-workers** mailing list](https://lists.perl.org/list/cpan-workers.html).
 
-[cpan-workers]: http://lists.perl.org/list/cpan-workers.html
+## Rules for editing content
 
-## Rules for editing content:
+- Don't add new output files.
+Only files already on cpan.org should be added (and then to their current location).
+Generally email [ask@perl.org](mailto:ask@perl.org) to get new files included here.
+Making new templates/scripts/etc to produce the output files is fine.
 
-* Don't add new output files. Only files already on cpan.org should be
-added (and then to their current location). Generally email
-ask@perl.org to get new files included here. Making new
-templates/scripts/etc to produce the output files is fine.
+- Be conservative in your edits.
+While lots of updates are needed, this has been around for 20 years and will be for many more.
+There's no rush.
 
-* Be conservative in your edits.  While lots of updates are needed,
-this has been around for 20 years and will be for many more. There's
-no rush.
-
-* No style edits for now. Cleanups yes; but a new design/layout will
-wait.
+- No style edits for now.
+Cleanups yes; but a new design/layout will wait.
 
 ## How to submit changes
 
-A 'pull request' on github is the best way. Sending a patch to the
-cpan-workers list at the same time will be a good way to get peer
-review. A change with a few "+1" votes from the list are more likely
-to be expediently pulled in.
+A 'pull request' on GitHub is the best way.
+Sending a patch to the [cpan-workers](mailto:cpan-workers@perl.org) list at the same time will be a good way to get peer review.
+A change with a few "+1" votes from the list is more likely to be expediently pulled in.
 
 ## How it works
 
-Install Template Toolkit and the other requirements with `cpanm` by running `make install`.
+Install [Template Toolkit](https://metacpan.org/dist/Template-Toolkit) and the other requirements with `cpanm` by running `make install`.
 
 To fetch the data needed for the site, run `make update-data update-daily`.
 
-Then run `make`. This will in turn run `ttree` and generate output
-files in the html/ directory.
+Then run `make`.
+
+This will in turn run [`ttree`](https://metacpan.org/release/ABW/Template-Toolkit-2.00/view/bin/ttree) and generate output files in the `html/` directory.
```

Comment:
I didn't know what `ttree` is until I submitted this PR and researched a bit more on the utility.